mcs/class/System/Test/System.Diagnostics/SourceSwitchTest.cs

```diff
@@ -53,7 +53,7 @@ public class SourceSwitchTest
 		public void ConstructorNullName ()
 		{
 			SourceSwitch s = new SourceSwitch (null);
-			AssertHelper.IsEmpty (s.DisplayName);
+			Assert.AreEqual (s.DisplayName.Length, 0);
```

Comment:
@spouliot that's actually what AssertHelper.IsEmpty is doing now since we switched to nunitlite: https://github.com/mono/mono/blob/695282c94ba45c27f909e843ca2b7a86c7bd085a/mcs/class/test-helpers/NunitHelpers.cs#L67-L70

I'm a bit surprised these cause problems for XM since I thought guiunit is nunitlite based too?